elements/pfe-cta/src/pfe-cta.story.js

```diff
@@ -23,14 +23,13 @@ stories.add(PfeCta.tag, () => {
     priority: {
       title: "Priority",
       type: "string",
-      enum: ["default", "primary", "secondary"],
+      enum: ["primary", "secondary"],
```

Comment:
@castastrophe - Why are we getting rid of "default"?

Reply:
We're not getting rid of default, we're letting the storybook util add in a default value automatically instead of hardcoding a value of default (which is technically not a valid input for that attribute anyway).

Reply:
There was an issue with the util function that wasn't injecting the default value consistently so I just updated that with my last commit!  Thanks for noting this.